src/plotman/csv_exporter.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import sys
+from dateutil.parser import parse as parse_date
+from plotman.log_parser import PlotLogParser
+from plotman.plotinfo import PlotInfo
+
+def export(logfilenames, save_to = None):
```

Comment:
I would probably tend to leave the file opening to the caller instead of having these small 'helpers'.  It just generally seems better to leave the file handling explicit at the higher levels.